jupyterlab_gcpscheduler/src/components/dialog.tsx

```diff
@@ -170,54 +158,21 @@ export class SchedulerDialog extends React.Component<Props, State> {
   }
 
   render() {
-    const projectId = this.getProjectId();
     return (
       <Dialog open={this._isOpen()} scroll="body">
         {this.state.showCreateForm && (
           <header className={localStyles.header}>
             <span className={localStyles.title}>
-              Create a notebook{' '}
-              {this.state.creatingExecution ? 'execution' : 'schedule'}{' '}
+              Submit notebooks to Executor
               <Badge value="alpha" />
             </span>
-            <IconButtonMenu
```

Comment:
Forgot to comment on this this morning, is there a reason we're moving out all of this? I know in the past the Reset configuration was useful to clear out settings that might have mistakenly been saved and this would also drop the View link?

Reply:
Huaiwei asked me if we could drop it. I can add it back if you feel it's useful.

Reply:
I'm fine with dropping the links to the Pantheon pages, but for now let's leave the Reset settings since it's an easy way to clear out anything that might have been previously saved that might be affecting the initial dialog state.

Reply:
That's fair. Will add it back